server/config/config.go

```diff
@@ -306,3 +309,11 @@ func GetConfig(v *viper.Viper) Config {
 		},
 	}
 }
+
+func (c Config) ValidateBasic() error {
```

Comment:
Same here

Reply:
ACK. Everything in a publicly exposed API should be documented.

Reply:
Done!